pkg/memory-monitor/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM lfedge/eve-alpine:1f7685f95a475c6bbe682f0b976f12180b6c8726 as memory-monitor-build
+
+ENV BUILD_PKGS gcc musl-dev make linux-headers cmake build-base
+ENV PKGS alpine-baselayout musl-utils
+RUN eve-alpine-deploy.sh
+
+COPY . /root/memory-monitor
+
+WORKDIR /root/memory-monitor
+
+RUN make dist
+RUN chmod +x /root/memory-monitor/dist/memory-monitor-handler.sh
+
+FROM scratch
+WORKDIR /
+# Create base Linux layout, copying from /out (produced by eve-alpine-deploy.sh)
+COPY --from=memory-monitor-build /out/ /
+# Copy all the memory-monitor artifacts
+COPY --from=memory-monitor-build /root/memory-monitor/dist/memory-monitor /sbin/
+COPY --from=memory-monitor-build /root/memory-monitor/dist/memory-monitor-handler.sh /sbin/
+COPY --from=memory-monitor-build /root/memory-monitor/dist/memory-monitor.conf /etc/
+
+# TODO: Later we will use apparmor to confine the memory-monitor. The apparmor profile template is in dist directory.
+# COPY --from=memory-monitor-build /root/memory-monitor/dist/sbin.memory-monitor-handler /etc/apparmor.d/
```

Comment:
The apparmor project goes into `pkg/apparmor/profiles`.

Reply:
Ahaaa... Good to know, thanks!

Reply:
Do you think we should add a proper profile with this PR? In this case, I'll try to test it, but there is a chance I'll need some help. Could you review the current template?
https://github.com/OhmSpectator/eve/blob/feature/integrate-memory-monitor/pkg/memory-monitor/src/monitor/sbin.memory-monitor-handler
Also worth noting is that the AppArmor profile exists only for the handler script but not for the monitor itself. Do we need both, or can we somehow combine them?